.circleci/config.yml

```diff
@@ -330,6 +330,14 @@ commands:
 #        JOBS
 # -------------------------
 jobs:
+  wait_until_previous_workflow_done:
+    executor: workbench-executor
+    steps:
+      - run:
+          working_directory: ~/workbench
+          command: bash .circleci/workflow_queue-v1.sh
```

Comment:
New `workflow-queue-v1.sh` script: Use CircleCI `v1` api and project token in `CIRCLECI_API_TOKEN` env variable.

Reply:
Please also delete your personal API token from the env vars if it's still there (I don't remember if you used the same variable name in the previous version)

Reply:
👍🏻 Deleted.